lib/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ rest/models/edit_message_args.cpp rest/models/bulk_delete_message_args.cpp
 rest/models/edit_channel_permissions_args.cpp rest/models/create_channel_invite_args.cpp
 rest/models/add_dm_recipient_args.cpp)
 
-include_directories(${CMAKE_SOURCE_DIR}/include)
+include_directories(${CMAKE_SOURCE_DIR}/include ${CMAKE_SOURCE_DIR}/include/disccord)
```

Comment:
im only replacing them for the `.hpp` files, not the `.cpp` files since they dont need to have the dependency of matching up with how the includes are installed. so this included thing prevents me to have to go through all of them as well and change to `disccord/whatever`